client/my-sites/email/email-management/titan-manage-mailboxes/index.jsx

```diff
@@ -134,7 +134,7 @@ class TitanManageMailboxes extends Component {
 
 						<VerticalNavItemEnhanced
 							description={ translate(
-								'Download our Android and iOS apps to access your emails on the go'
+								"Download Titan's Android and iOS apps to access your emails on the go"
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Access your email on the go with Titan's Android and iOS apps.' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/gsuite-sidebar-warning)